lib/utils/compose.coffee

```diff
@@ -334,6 +334,7 @@ tagServiceImages = (docker, images, serviceImages) ->
 		serviceImage = serviceImages[d.serviceName]
 		imageName = serviceImage.is_stored_at__image_location
 		[ _match, registry, repo, tag = 'latest' ] = /(.*?)\/(.*?)(?::([^/]*))?$/.exec(imageName)
+		_match = _match  # to dismiss a linter warning: Variable is assigned to but never read
```

Comment:
I think you can write something like this:
```typescript
[ , registry, repo, tag = 'latest' ] = /(.*?)\/(.*?)(?::([^/]*))?$/.exec(imageName)
```

Reply:
:+1: @pdcastro I'm never able to find the coffelint docs for these stuff, since their [docs website](www.coffeelint.org) isn't accessible and I always end up searching in their sources.
TIL that they also support `disable-next-line` :)